tools/shoestring/tests/commands/test_renew_certificates.py

```diff
@@ -82,7 +82,7 @@ async def test_can_renew_node_certificate():
 
 
 async def test_can_renew_node_certificate_with_ca_password():
-	await _assert_can_renew_node_certificate('abc')
+	await _assert_can_renew_node_certificate('abcd')
```

Comment:
why this change? is password limit different?

Reply:
yea, the password limit is different between versions.